devnotes/pnor_layout.md

```diff
@@ -0,0 +1,181 @@
+# About this document
+This document gathers known informations about partition layout in the original
+PNOR image from the Talos II platform.
+
+# Partition table
+
+PNOR contains partition table at offset 0. It is constructed out of header and multiple entries.
+Refer to the [ffs source code](https://github.com/open-power/ffs/blob/3ec70fbc458e32eef0d0b1de79688b4dc48cbd57/ffs/ffs.h#L108) for more detailed informations.
```

Comment:
@IgorBagnucki Pleas fix line wrap. `[ffs source code](https://github.com/open-power/ffs/blob/3ec70fbc458e32eef0d0b1de79688b4dc48cbd57/ffs/ffs.h#L108)` could be in a separate line

Reply:
Fixed